loader/src/main/scala/com/actian/spark_vector/loader/spark_sql_utils/SparkSqlTable.scala

```diff
@@ -0,0 +1,32 @@
+package com.actian.spark_vector.loader.spark_sql_utils
+
+import java.util.concurrent.atomic.AtomicLong
+import org.apache.spark.sql.DataFrame
+
+sealed trait SparkSqlTable {
+  def table_name : String
+  def quoted_name = sparkQuote(table_name)
+  def close() = {}
+}
+
+case class HiveTable(override val table_name: String) extends SparkSqlTable 
```

Comment:
better leave close as abstract in SparkSQLTable and force it to be defined in subclasses.